src/main/frontend/extensions/latex.cljs

```diff
@@ -58,17 +58,20 @@
        s])))
 
 (defn html-export
-  [s block? display?]
+  [s block? _display?]
   (let [element (if block?
                   :div.latex
                   :span.latex-inline)]
     [element (cond
                block?
                (util/format "$$%s$$" s)
 
+               ;; TODO: We never get to :else b/c this branch always evaluates as true
```

Comment:
@tiensonqin This may be a bug as it never reaches the last branch. Was `:display?` supposed to be the arg `display?`

Reply:
Another good catch! Yes, it should be `display?`.

Reply:
Just pushed a fix 32a8d98d0e93883c6c30c1f07fbdcaa22ce56417.